include/drivers/interrupt_controller/gic.h

```diff
@@ -252,9 +252,13 @@
 #define GIC_INTID_SPURIOUS		1023
 
 /* Fixme: update from platform specific define or dt */
-#define GIC_NUM_CPU_IF			1
+#define GIC_NUM_CPU_IF			CONFIG_MP_NUM_CPUS
 /* Fixme: arch support need to provide api/macro in SMP implementation */
+#ifdef CONFIG_ARM64
```

Comment:
```suggestion
#if defined(CONFIG_ARM64) && defined(CONFIG_SMP)
```

perhaps?

Reply:
No. There is a test that not have SMP, but need mutilple cpus. "tests/kernel/mp"